diff/diff.go

```diff
@@ -69,21 +61,13 @@ func ProcessDiffs(matcher lsearch.Matcher, hunk *diff.Hunk, flagsRef lflags.Flag
 		// only one for now
 		elementMatcher := matcher.Elements[0]
 		for _, flagKey := range elementMatcher.FindMatches(line) {
-			if _, ok := flagMap[op][flagKey]; !ok {
-				flagMap[op][flagKey] = make(lflags.AliasSet)
-			}
-			if aliasMatches := matcher.FindAliases(line, flagKey); len(aliasMatches) > 0 {
-				if _, ok := flagMap[op][flagKey]; !ok {
-					flagMap[op][flagKey] = make(lflags.AliasSet)
-				}
-				for _, alias := range aliasMatches {
-					flagMap[op][flagKey][alias] = true
-				}
-			}
+			aliasMatches := elementMatcher.FindAliases(line, flagKey)
```

Comment:
this was always supposed to be calling `elementMatcher.FindAliases` - since we don't have monorepo support, it hasn't mattered so far that it wasn't